### ISSUE TITLE
Utils.c - nrt_Utils_parseGeographicString function has issues when degrees value is zero

### ISSUE THREAD

**Issue by dfeng1** [mdaus/nitro#44]
In the nrt_Utils_parseGeographicString function, when the input location string has the degree evaluate to 0, the sign is going to be incorrect for 'S' and 'W'.

I think the problem might be here:
```
if ((degreeOffset == 2 && dir == 'S') || (degreeOffset == 3 && dir == 'W'))
{
    *degrees *= -1; // will not be correct sign when degrees is zero
}
```

**Comment by JonathanMeans** [comment]
Could you please confirm my understanding of how this is supposed to work?

    "123456W" -> D: -12, M: -34, S: -56.0
    "001234W" -> D: 0, M: -12, S: -34.0
    "000011W" -> D: 0, M: 0, S: -11.0

Or is the convention supposed to be that only the largest non-zero unit takes the sign?

**Comment by dfeng1** [comment]
I would expect something like 001234W when converted to decimal degrees to equal -0.2094444° instead of +0.2094444°. I don't know what needs to happen in the intermediate between string -> d,m,s -> decimal degrees. Negative minutes and seconds might not make sense but depends on how it's interpreted down the line.

**Comment by asylvest** [comment]
Resolved by #45
